subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -785,6 +786,9 @@ static int __wifi_args_to_params(const struct shell *sh, size_t argc, char *argv
 		case 'S':
 			params->suiteb_type = atoi(state->optarg);
 			break;
+		case 'T':
+			params->TLS_cipher = atoi(optarg);
```

Comment:
Please use `state->optarg`, same as other lines in this function

Reply:
Thanks. Code updated.